docs/overview/usage_patterns.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: i18n_page
+title: pages.usage_patterns
+parent: pages.understanding_firefly
+nav_order: 2
+---
+
+# Usage Patterns
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+There are usages mode for Hyperledger FireFly: **Web3 Gateway** and **Multiparty**
```

Comment:
Reads a little oddly, perhaps "There are two modes of usage for Hyperledger Firefly"